fontforgeexe/startui.c

```diff
@@ -632,22 +645,14 @@ static int splash_e_h(GWindow gw, GEvent *event) {
 	splash_cnt = 0;
       break;
       case et_timer:
-	if ( event->u.timer.timer==autosave_timer ) {
-	    DoAutoSaves();
-	} else if ( event->u.timer.timer==splasht ) {
-	    if ( ++splash_cnt==1 )
-		GDrawResize(gw,splashimage.u.image->width,splashimage.u.image->height-30);
-	    else if ( splash_cnt==2 )
-		GDrawResize(gw,splashimage.u.image->width,splashimage.u.image->height);
-	    else if ( splash_cnt>=7 ) {
-		GGadgetEndPopup();
-		GDrawSetVisible(gw,false);
-		GDrawCancelTimer(splasht);
-		splasht = NULL;
-	    }
-	} else {
```

Comment:
The aforementioned else clause.

Reply:
Resolved in https://github.com/fontforge/fontforge/pull/3754/commits/caf4608143151aadf8db1e20656a4ac87d8e649a